src/main.ts

```diff
@@ -96,7 +109,7 @@ export const playLevel = async (rawLevelUrl: string, videoName: string, folder:
 
   // To avoid chopping the end of the video prematurely, we will stop the video 1 second later, adjusted
   // for our tick rate time scaling
-  const tailWaitTimeMs = 1 * (defaultTickRate / tickRate) * 1000.0
+  const tailWaitTimeMs = 2.5 * (defaultTickRate / tickRate) * 1000.0
```

Comment:
Is the tailWait really necessary since once the game is completed we should get the value of T and expression

Reply:
For some reason this seems to be necessary to get a clean tail end of the video, without it it stops short.  I agree that this isn't the most elegant solution though, it's probably just sweeping an actual problem under the rug - but it seems to work well enough for now...